---a/net/minecraft/dispenser/IBlockSource.java
+++b/net/minecraft/dispenser/IBlockSource.java
@@ -4,8 +4,8 @@
 import net.minecraft.tileentity.TileEntity;
 import net.minecraft.util.BlockPos;
 
-public interface IBlockSource extends ILocatableSource
-{
+public interface IBlockSource extends ILocatableSource {
+
     double getX();
 
     double getY();
@@ -19,4 +19,4 @@
     int getBlockMetadata();
 
     TileEntity getBlockTileEntity();
-}
+}
